src/pages/NewChairConsole.js

```diff
@@ -1,40 +1,426 @@
+    return div({style: Object.assign({}, borderStyle, styles.TABLE.RECORD_ROW), key: `${election.frontEndId}`}, [
+      div({
+        style: Object.assign({}, styles.TABLE.DATA_ID_CELL, dataIDTextStyle),
+        onClick: (e) => openModal(election.referenceId),
+        isRendered: !isNil(election.frontEndId),
+        onMouseEnter: applyDarTextHover,
+        onMouseLeave: removeDarTextHover
+      }, [election.frontEndId]),
+      div({style: Object.assign({}, styles.TABLE.TITLE_CELL, recordTextStyle)}, [election.projectTitle]),
+      div({style: Object.assign({}, styles.TABLE.SUBMISSION_DATE_CELL, recordTextStyle)}, [formatDate(election.createDate)]),
+      div({style: Object.assign({}, styles.TABLE.DAC_CELL, recordTextStyle)}, [election.dac.name]),
+      div({style: Object.assign({}, styles.TABLE.ELECTION_STATUS_CELL, recordTextStyle)}, [election.electionStatus]),
+      div({style: Object.assign({}, styles.TABLE.ELECTION_ACTIONS_CELL, recordTextStyle)}, ['Placeholder for buttons. (font style is placeholder as well)'])
+    ]);
+  });
+};
+
+const ModalDetailRow = (props) => {
+  return (
+    div({style: styles.MODAL.DAR_DETAIL_ROW}, [
+      div({style: styles.MODAL.DAR_LABEL}, [props.label]),
+      div({style: styles.MODAL.DAR_DETAIL}, [props.detail])
+    ])
+  );
+};
+
+const returnPIName = (havePI, isThePI, displayName, piName) => {
+  let returnName;
+  if(isThePI === "true") {
+    returnName = displayName;
+  }else if(havePI === "true") {
+    returnName = piName;
+  }
+  return returnName || '- -';
+};
+
+const NewChairConsole = (props) => {
+  const [showModal, setShowModal] = useState(false);
+  // const [darLimit, setDarLimit] = useState(5);
+  // const [currentPage, setCurrentPage] = useState(1);
+  // const [totalPendingVotes, setTotalPendingVotes] = useState(0);
+  // const [filteredList, setFilteredList] = useState([]);
+  // const [visibleList, setVisibleList] = useState([]);
+  const [electionList, setElectionList] = useState([]);
+  const [darDetails, setDarDetails] = useState({});
+  const [detailLoadingStatus, setDetailLoadingStatus] = useState(false);
+  const [researcherDetails, setResearcherDetails] = useState({});
+  const darCache = useRef({});
+
+  //NOTE: skeleton function for pagination feature, to be implemented/expanded/revised in later PR
+  // const goToPage = useCallback((targetPage, currentPage, filteredList, darLimit) => {
+  //   const targetStartIndex = targetPage * currentPage - darLimit;
+  //   if(targetStartIndex + 1 < filteredList.length) {
+  //     const targetEndIndex = targetPage * currentPage;
+  //     setCurrentPage(targetPage);
+  //     setVisibleList(filteredList.slice(targetStartIndex, targetEndIndex));
+  //   }
+  // }, [setCurrentPage, setVisibleList]);
+
+  useEffect(() => {
+    const init = async() => {
+      try {
+        const currentUser = Storage.getCurrentUser();
+        const caseList = await PendingCases.findDataRequestPendingCasesByUser(currentUser.dacUserId);
```

Comment:
This is using the old API, not the v2 api.

Reply:
What's the v2 variant for pendingCases? Is it just using the standard DAR get and then filtering it out manually?